### ISSUE TITLE
ChefSpec deprecation warnings in libraries/matchers.rb

### ISSUE THREAD

**Issue by reijop** [opscode-cookbooks/windows#131]
Running ChefSpec on a cookbook that includes this cookbook causes deprecation warning spam like so:

```
[DEPRECATION] `ChefSpec::Runner.define_runner_method' is deprecated. Please use `ChefSpec.define_matcher' instead.
```

**Comment by JonathanPorta** [comment]
@mwrock Hey! Looks like the relevant PR for this we merged in and released. Can this issue be closed?

**Comment by mwrock** [comment]
Sounds good

**Comment by btm** [comment]
Fixed by #132
